libs/server/repository/core/application-services/src/lib/queries/get-reviewer-statistics/get-reviewer-statistics.handler.ts

```diff
@@ -39,6 +39,10 @@ export class GetReviewerStatisticsHandler
       )
     );
 
+    repositoryStatistics = repositoryStatistics.filter(
```

Comment:
It is crucial to stay the node application very optimized. Please filter empty prs somewhere near line 38. Before statistics object is created.

Reply:
Could you please verify if current solution is good enough?  Thanks